src/sentry/snuba/spans_rpc.py

```diff
@@ -40,116 +32,6 @@ def get_resolver(params: SnubaParams, config: SearchResolverConfig) -> SearchRes
     )
 
 
```

Comment:
Could you keep `run_table_query` in here, but it auto fills the resolver before calling the common? Seems strange that the way to call spans.table is to call the common function instead